Gallery website/gallery.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html class="gallery">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="style.css" media="screen"/>
+</head>
+<body>
+
+  <h2>G A L L E R Y</h2>
+<div class=container>
```

Comment:
Maybe you could fix the indentation below the `<body>` tag by moving everything to the right because your `<div>` tag is spaced differently than the` <h2>` tag above it.